docs/integration-guides/key-management.md

```diff
@@ -454,6 +454,9 @@ curl -d '{
 }
 ```
 
+!!! tip "Use block subtype as a sanity check"
+    Since V18.0, [`process`](/commands/rpc-protocol/#process) has an optional string `"subtype"`, which takes the value of send/receive/open/change. This field can be used to prevent performing an unintended operation with a block.
```

Comment:
consider adding more explanation "This field can be used to prevent performing an unintended operation with a block as the request will return an error if the block details don't match up with the provided `"subtype"`"